GitInfo.txt

```diff
@@ -1 +1 @@
-6.0.100-preview.3
+6.0.100-preview.4
```

Comment:
Is there something else I should do here to reset the commit distance for the version number?

Reply:
Nop , we use that as  the final version number

Reply:
i think we would this on the release branch no?